src/themes/dark.scss

```diff
@@ -69,4 +69,5 @@
   .wxc-button { display: flex; }
   .wxc-old-icon { display: none; }
   .wxc-icon { display: block; }
+  .modal-header .old-close-icon { display: none; }
```

Comment:
`.wxc-modal .old-close-icon`

all css rules must have at least one prefixed (`.wxc-`) css class

Reply:
.old-close-button would be a better name

Reply:
you also have to hide the new close button on legacy, but if it doesn't show because the icon does not show, it's fine